app/components/UI/TooltipButton.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box, Text } from 'rebass'
+
+const StyledTooltipIconBox = styled(Box)`
+  border-radius: 50%;
+  height: 16px;
```

Comment:
I think this should be slightly smaller. In the mockup it is 13px.

Reply:
I made it 13px at first but it looked small to me. I will change it to 13px.